# -----------------------------------------------------------------------------------
# Dynamic Movement Primitive (DMP)
# related parameters used in dmp_parameters.cpp
# -----------------------------------------------------------------------------------

dmp:

  execution_duration: 1.0
  can_sys_cutoff: 0.001

  k_gain_icra2009: 50.0

  alpha_z_nips2003: 10.0

#  beta_can: 400.0
#  beta_trans: 400.0
#  beta_can: 0.0
#  beta_trans: 0.0
#  lower_beta: 0.04
#  upper_beta: 0.08
#  type: 0